docs/examples/index.md

```
@@ -0,0 +1,103 @@
+    conn = await create_connection()
+    try:
+        yield DatabaseState(connection=conn)
+    finally:
+        await conn.close()
+
+async def main():
+    async with ctx.scope("app", disposables=(database_connection(),)):
+        # Database connection is available here
+        db = ctx.state(DatabaseState)
+        # Connection is automatically closed when scope exits
+```
```

markdownlint-cli2 (0.17.2)

Files should end with a single newline character
null

(MD047, single-trailing-newline)